docs/Config.md

```diff
@@ -4,14 +4,18 @@
 
 The location of the user config will differ depending on your OS. You can open it via lazydocker by opening the application, clicking on the 'project' panel at the top left and pressing 'o' (or pressing 'e' if your files open in vim).
 
-Changes to the user config will only take place after closing and re-opening lazydocker
+Changes to the user config will only take place after closing and re-opening lazydocker.
 
 ### Locations:
 
-- OSX: `~/Library/Application Support/jesseduffield/lazydocker/config.yml`
-- Linux: `~/.config/lazydocker/config.yml`
+Lazydocker follows the [XDG base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) meaning that the config will be placed in `XDG_CONFIG_HOME` if the variable is found.
```

Comment:
maybe rephrase?
`Lazydocker adheres to the XDG Base Directory Specification, placing its configuration in XDG_CONFIG_HOME if the variable is set.`